Sources/Commands/SwiftTool.swift

```diff
@@ -366,6 +366,11 @@ public class SwiftTool<Options: ToolOptions> {
             option: parser.add(option: "--trace-resolver", kind: Bool.self),
             to: { $0.enableResolverTrace = $1 })
 
+        binder.bind(
+            option: parser.add(option: "--jobs", shortName: "-j", kind: Int.self,
+                usage: "Limit the number of parallel jobs that we tell llbuild to perform."),
```

Comment:
llbuild is an implementation detail. How about "The number of jobs to spawn in parallel during the build process"?

Reply:
I agree, but wasn't sure exactly what to put there. How about `Limit the number of parallel jobs that we perform.` ?

Reply:
Do you not like "The number of jobs to spawn in parallel during the build process"?

Reply:
That works for me! Sorry, I just woke up 🤦.

Reply:
Great! Can you remove the full stop (.) ? We don't use it in the CLI help.